<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomQwtPlot</class>
 <widget class="QWidget" name="CustomQwtPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>21</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>X=</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtAvg">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>SD=</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtSd">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>31</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>CV%=</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtCV">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>10</y>
        <width>50</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>31</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>6σ=</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt6sigma">
      <property name="geometry">
       <rect>
        <x>343</x>
        <y>11</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDataTable">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>7</y>
        <width>60</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数据表</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>35</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>X=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>35</y>
        <width>51</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>SD=</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCustAvg">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCustSD">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CV%=</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtCustCV">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>34</y>
        <width>50</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>起止日期:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>537</x>
        <y>10</y>
        <width>51</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>至</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>31</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>n=</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="IfCaculate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>计算值</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="IfCustum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>32</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>自定义</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="txtCust6sigma">
      <property name="geometry">
       <rect>
        <x>343</x>
        <y>34</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>33</y>
        <width>31</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>6σ=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCaculate">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>7</y>
        <width>60</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>累积计算</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="inputStartDate">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>5</y>
        <width>85</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="inputEndDate">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>5</y>
        <width>85</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInit">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>7</y>
        <width>60</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>初始化</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtN">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>7</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QLabel" name="txtTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>421</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>[title]</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QwtPlot" name="qwtPlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_footer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QLabel" name="txtFooter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>[footer]</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
